ui/apps/platform/src/routePaths.ts

```diff
@@ -422,6 +427,7 @@ export const basePathToLabelMap = {
     [configManagementPath]: 'Configuration Management',
     [riskBasePath]: 'Risk',
     [apidocsPath]: 'API Reference',
+    [apidocsPathV2]: 'API Reference(v2)',
```

Comment:
@charmik-redhat 
Thanks for adding the screenshot.

I think we should explicit label the link for the existing v1 API as "(v1)" in the menu link. (and also have a space between the name and version)

```suggestion
    [apidocsPath]: 'API Reference (v2)',
    [apidocsPathV2]: 'API Reference (v2)',
```

Reply:
But leave the original URL path the same, because we will go back to one URL when we put a version selector to collapse it back to one page.

Reply:
Thanks, that sounds better. Updated the text to `API Reference (v1)` and `API Reference (v2)`